tests/pass/shims/x86/intrinsics-x86-bmi.rs

```diff
@@ -30,6 +35,16 @@ unsafe fn test_bmi_32() {
             assert_eq!(_bextr_u32(u32::MAX, i, 4), 0b1111);
             assert_eq!(_bextr_u32(u32::MAX, 4, i), (1 << i) - 1);
         }
+
+        // Ensure that indices larger than the bit count are covered.
+
+        for i in 0..1000 {
+            assert_eq!(_bextr_u32(u32::MAX, 0, i).count_ones(), (i & 0xff).min(32));
```

Comment:
Why `(i & 0xff).min(32)`...?

Reply:
The instruction only takes the lowest eight bits to generate the index, thus the mask. If the index is out of bounds, the original input won't be changed, thus the `min(32)`.

Reply:
Okay, that seems worth a short comment.

Reply:
Done.